Wordle.Api/Wordle.Api/Dtos/MoreWordsDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Wordle.Api.Models;
+
+namespace Wordle.Api.Dtos;
+
+public class MoreWordsDto
```

Comment:
```suggestion
public class PagedWordsDto
```
This is a better name for this in my opinion.

Reply:
Thank you for the suggestion! I agree.